tests/jsmarionette/client/marionette-client/lib/marionette/client.js

```diff
@@ -473,11 +454,10 @@
       var args = Array.prototype.slice.call(arguments),
           callback = args.shift();
 
-      if (!callback) {
-        callback = this.defaultCallback;
-      }
+      if (!cb)
+        cb = this.defaultCallback;
 
-      assert(typeof callback === 'function', 'you must use functions');
```

Comment:
`===` is intentional here. See http://bonsaiden.github.io/JavaScript-Garden/#types.equality